SpatialGDK/Source/SpatialGDKEditor/Public/SpatialGDKEditorSettings.h

```diff
@@ -431,6 +431,17 @@ class SPATIALGDKEDITOR_API USpatialGDKEditorSettings : public UObject
 	UPROPERTY(EditAnywhere, config, Category = "Cloud Connection")
 	FString DevelopmentDeploymentToConnect;
 
+	/** Whether to start local server worker when connecting to cloud deployment. If selected, make sure that the cloud deployment you want to connect to is not automatically launching Server-workers. (That your workers are "manual_connection_only" type)*/
+	UPROPERTY(EditAnywhere, config, Category = "Cloud Connection", meta = (DisplayName = "Connect local server worker to the cloud deployment"))
+	bool bConnectServerToCloud;
+
+	/** Port on which the receptionist proxy will be available.*/
+	UPROPERTY(EditAnywhere, config, Category = "Cloud Connection", meta = (EditCondition = "bStartLocalServerWorker", DisplayName = "Local Receptionist Port"))
+	int32 LocalReceptionistPort = SpatialConstants::DEFAULT_SERVER_RECEPTIONIST_PROXY_PORT;
```

Comment:
I believe these are already initialized in the constructor.
```suggestion
	int32 LocalReceptionistPort;
```